medusa/post_processor.py

```diff
@@ -510,51 +510,36 @@ def _add_to_anidb_mylist(self, file_path):
 
     def _find_info(self):
         """
-        For a given file try to find the showid, season, and episode.
+        For a given file try to find the show, season, epsiodes, version and quality.
 
-        :return: A (show, season, episodes, quality, version) tuple
+        :return: A (show, season, episodes, version, quality) tuple
         """
-        show = season = quality = version = None
+        show = season = version = quality = None
```

Comment:
As long as you don't assign a mutable type, it's okay.
So x = y = z = None, is okay.
x = y = z = [], is dangerous.